core/platform/models.py

```diff
@@ -33,6 +33,10 @@
     'question', 'recommendations', 'skill', 'statistics', 'story', 'suggestion',
     'topic', 'user')
 
+NAMES_THAT_CAN_BE_PSEUDONYMIZED = utils.create_enum(  # pylint: disable=invalid-name
```

Comment:
MODULES_WITH_PSEUDONYMIZABLE_CLASSES or MODULES_WITH_CLASSES_THAT_CAN_BE_PSEUDONYMIZED

("NAMES" is vague)